README.md

```diff
@@ -47,7 +46,7 @@
 | [Koishi](https://koishi.chat/) | Koishi Team | TypeScript | Chatbot framework based on Satori |
 | [Avilla](https://graia.cn/avilla/) w/ [avilla-satori](https://github.com/GraiaProject/Avilla/tree/ryanvk-v1.2/avilla/satori) | Avilla [@RF-Tar-Railt](https://github.com/RF-Tar-Railt) | Python | The next-gen Python framework for IM development |
 | [NoneBot](https://nonebot.dev/) w/ [adapter-satori](https://github.com/nonebot/adapter-satori) | NoneBot [@RF-Tar-Railt](https://github.com/RF-Tar-Railt) | Python | Python cross-platform bot framework |
-| [Yutori-QQ](https://github.com/Nyayurn/Yutori-QQ) or<br>[Yutori-QQ-SpringBootStarter](https://github.com/Nyayurn/Yutori-QQ-SpringBootStarter) | [@Nyayurn](https://github.com/Nyayurn) | Java | Java **QQ** bot framework SDK based on Satori Protocol |
+| [Yutori](https://github.com/Nyayurn/Yutori) | [@Nyayurn](https://github.com/Nyayurn) | Kotlin | Java bot framework SDK based on Satori Protocol |
```

Comment:
```suggestion
| [Yutori](https://github.com/Nyayurn/Yutori) | [Nyayurn](https://github.com/Nyayurn) | Kotlin | Java bot framework SDK based on Satori Protocol |
```

You don't need to put `@` in front of the author's name if you or your project aren't on the team.